library-backend/library-api/src/main/java/org/internship/library/api/Book.java

```diff
@@ -1,69 +1,43 @@
 package org.internship.library.api;
 
 /**
- * Book class
+ * This interface defines the structure of a Book
  */
-public class Book {
-    private Long id;
-    private String title;
-    private String author;
-    private Integer numberOfPages;
+public interface Book {
+    /**
+     * Setter and getter for id
+     *
+     * @return
+     */
+    public String getId();
+
+    public void setId(String id);
+
+    /**
+     * Setter and getter for title
+     *
+     * @return
+     */
+    public String getTitle();
+
+    public void setTitle(String title);
+
+    /**
+     * Setter and getter for author
```

Comment:
The javadoc is per method. You've only added it to the Getters.

Please add for each method and for each method add the right description.